ballerina/build.gradle

```diff
@@ -57,6 +57,7 @@ ballerina {
     packageOrganization = packageOrg
     module = packageName
     langVersion = ballerinaLangVersion
+    testCoverageParam = "--code-coverage --coverage-format=xml --includes=io.ballerina.stdlib.io.*:ballerina.io*"
```

Comment:
Shall we bump the plugin version to `0.10.0`? This should be addressed in the latest version

Reply:
Ah right. Will do.